### PR TITLE
DOCS: 4x updates to the general doc

### PR DESCRIPTION
**Pull request by ljamen**
### Documentation

Updates to general doc introductions for 4x.

### CODE REVIEW

File: docs/about/doc_overview.adoc [hunk 1]
```diff
@@ -124,11 +128,18 @@ xref:{rootdir}/mp/guides/overview.adoc[MP Guides]
...
-Learn:  
+ 
+
+Training:  
  https://learn.oracle.com/ols/learning-path/become-a-helidon-microservices-developer-professional/88258/114512[Become a Helidon Microservices Developer Professional]
  
 Certification: 
   https://learn.oracle.com/ols/learning-path/become-a-helidon-microservices-developer-professional/88258/114512[Helidon Microservices Developer Professional (1Z0-1113)]
+  
+New to Helidon? Check out https://a.co/d/0niQnwC[Beginning Helidon] to learn more about building cloud-native microservices with Helidon.
```
Comment (dalexandrov) [inline docs/about/doc_overview.adoc:139]: I have another PR for book integration https://github.com/helidon-io/helidon/pull/7577
Maybe reuse mine?

File: docs/about/intro.adoc [hunk 2]
```diff
@@ -27,12 +27,12 @@
 
 * Introduction of the new Helidon WebServer (Project N&iacute;ma): a virtual threads-based web server implementation based on https://openjdk.org/jeps/444[JDK Project Loom]  virtual threads.
 
-* Removed Helidon's Reactive WebServer and WebClient that were based on Netty. Our new implementations are based on virtual threads that have a blocking style API (Helidon N&iacute;ma). Learn more: <<Helidon 4 WebServer, Helidon 4 WebServer.>>
+* Removed Helidon's Reactive WebServer and WebClient that were based on Netty. Our new implementations are based on virtual threads that have a blocking style API (Project N&iacute;ma). Learn more: <<Helidon 4 WebServer, Helidon 4 WebServer.>>
```
Comment (dalexandrov) [inline docs/about/intro.adoc:30]: Maybe: previously known as "Project Níma" ?

File: docs/about/intro.adoc [hunk 3]
```diff
@@ -68,7 +68,7 @@ doSomething(request.content().as(JsonObject.class), response);
 
 
 == Helidon MP
-Helidon MP is Helidon's MicroProfile implementation and in Helidon 4 it supports MicroProfile 6 and the Jakarta EE 10 Core Profile. Your Helidon 3 MicroProfile application should migrate to Helidon 4 fairly easily (the most significant changes are in MicroProfile Metrics 5.0), and since Helidon's MicroProfile server is based on the new Helidon N&iacute;ma WebServer, you get all the benefits of running on virtual threads.
+Helidon MP is Helidon's MicroProfile implementation and in Helidon 4 it supports MicroProfile 6 and the Jakarta EE 10 Core Profile. Your Helidon 3 MicroProfile application should migrate to Helidon 4 fairly easily (the most significant changes are in MicroProfile Metrics 5.0), and since Helidon's MicroProfile server is based on the new Helidon WebServer (Project N&iacute;ma), you get all the benefits of running on virtual threads.
```
Comment (dalexandrov) [inline docs/about/intro.adoc:71]: Same as above.

File: docs/about/intro.adoc (hunk 2 above)
Comment (ljamen) [reply docs/about/intro.adoc:30]: I discussed this with Joe, Ed and Dmitry and they requested I use Project Nima just in this intro and then refer to it as the Helidon WebServer everywhere else.

**Review by ljamen — COMMENTED** [review]
@dalexandrov I updated the language in the overview of this intro to reflect the changes for SE in v4x. Please review.

File: docs/about/doc_overview.adoc [hunk 4]
```diff
@@ -124,11 +128,18 @@ xref:{rootdir}/mp/guides/overview.adoc[MP Guides]
...
 --
-Learn:  
+ 
+
+Training:  
  https://learn.oracle.com/ols/learning-path/become-a-helidon-microservices-developer-professional/88258/114512[Become a Helidon Microservices Developer Professional]
  
 Certification: 
   https://learn.oracle.com/ols/learning-path/become-a-helidon-microservices-developer-professional/88258/114512[Helidon Microservices Developer Professional (1Z0-1113)]
+
```
[the remark this answers is not in the document]
Comment (dalexandrov) [reply docs/about/doc_overview.adoc:138]: Yes :)

File: docs/about/intro.adoc (hunk 2 above)
Comment (dalexandrov) [reply docs/about/intro.adoc:30]: Ok

**Pull request merged**